src/main/resources/application.yml

```diff
@@ -47,6 +47,7 @@ case:
   creator:
     case-service: http://localhost:8082
     workflow-service: http://localhost:8091
+    document-service: http://localhost:8083
```

Comment:
This will need a helm chart change to add these to the deployment.

https://github.com/UKHomeOffice/hocs-helm-charts/blob/7cfc67f397a06707260e061d81d94c60d0e009a3/charts/hocs-case-migrator/templates/_envs.tpl#L22